### ISSUE TITLE
Runtime error on search.

### ISSUE THREAD

**Issue by lukevers** [alexander-bauer/distru#61]
``` log
2012/11/12 15:59:41 Distru version 0.8.3
2012/11/12 15:59:41 Configuration status:
    Generated in:    0.8.2 
    Indexers:    2 
    AutoIndexing:    1 
    Resouces:    2 
    Sites indexed:   1
2012/11/12 15:59:41 indexer> adding "mesh.lukevers.com"
2012/11/12 15:59:41 Started server on port 9049.
2012/11/12 15:59:41 Started webserver on port 9048.
2012/11/12 15:59:47 indexer> added "mesh.lukevers.com"
2012/11/12 15:59:48 <-[fc2e:9943:1633:403e:2346:3704:8cd8:1c78]:52514> searching "derp"
2012/11/12 15:59:52 http: panic serving [fc2e:9943:1633:403e:2346:3704:8cd8:1c78]:52514: runtime error: invalid memory address or nil pointer dereference
/usr/local/Cellar/go/1.0.3/src/pkg/net/http/server.go:589 (0x70935)
    _func_004: buf.Write(debug.Stack())
/usr/local/Cellar/go/1.0.3/src/pkg/runtime/proc.c:1443 (0x15f45)
    panic: reflect·call(d->fn, d->args, d->siz);
/usr/local/Cellar/go/1.0.3/src/pkg/runtime/runtime.c:128 (0x16a11)
    panicstring: runtime·panic(err);
/usr/local/Cellar/go/1.0.3/src/pkg/runtime/thread_darwin.c:418 (0x19b60)
    sigpanic: runtime·panicstring("invalid memory address or nil pointer dereference");
/Users/node/distru/index.go:50 (0x255c)
    (*Index).MergeRemote: defer conn.Close()
/Users/node/distru/search.go:14 (0x4d64)
    (*config).Search: index.MergeRemote(conf.Resources[i], true, conf.ResTimeout)
/Users/node/distru/web.go:25 (0x5d8f)
    searchHandler: results := Conf.Search(strings.Split(searchTerm, " "))
/usr/local/Cellar/go/1.0.3/src/pkg/net/http/server.go:703 (0x6477e)
    HandlerFunc.ServeHTTP: f(w, r)
/usr/local/Cellar/go/1.0.3/src/pkg/net/http/server.go:941 (0x655d2)
    (*ServeMux).ServeHTTP: mux.handler(r).ServeHTTP(w, r)
/usr/local/Cellar/go/1.0.3/src/pkg/net/http/server.go:669 (0x64591)
    (*conn).serve: handler.ServeHTTP(w, w.req)
/usr/local/Cellar/go/1.0.3/src/pkg/runtime/proc.c:271 (0x1404b)
    goexit: runtime·goexit(void)
2012/11/12 15:59:52 <-[fc2e:9943:1633:403e:2346:3704:8cd8:1c78]:52515> searching "derp"
2012/11/12 15:59:56 http: panic serving [fc2e:9943:1633:403e:2346:3704:8cd8:1c78]:52515: runtime error: invalid memory address or nil pointer dereference
/usr/local/Cellar/go/1.0.3/src/pkg/net/http/server.go:589 (0x70935)
    _func_004: buf.Write(debug.Stack())
/usr/local/Cellar/go/1.0.3/src/pkg/runtime/proc.c:1443 (0x15f45)
    panic: reflect·call(d->fn, d->args, d->siz);
/usr/local/Cellar/go/1.0.3/src/pkg/runtime/runtime.c:128 (0x16a11)
    panicstring: runtime·panic(err);
/usr/local/Cellar/go/1.0.3/src/pkg/runtime/thread_darwin.c:418 (0x19b60)
    sigpanic: runtime·panicstring("invalid memory address or nil pointer dereference");
/Users/node/distru/index.go:50 (0x255c)
    (*Index).MergeRemote: defer conn.Close()
/Users/node/distru/search.go:14 (0x4d64)
    (*config).Search: index.MergeRemote(conf.Resources[i], true, conf.ResTimeout)
/Users/node/distru/web.go:25 (0x5d8f)
    searchHandler: results := Conf.Search(strings.Split(searchTerm, " "))
/usr/local/Cellar/go/1.0.3/src/pkg/net/http/server.go:703 (0x6477e)
    HandlerFunc.ServeHTTP: f(w, r)
/usr/local/Cellar/go/1.0.3/src/pkg/net/http/server.go:941 (0x655d2)
    (*ServeMux).ServeHTTP: mux.handler(r).ServeHTTP(w, r)
/usr/local/Cellar/go/1.0.3/src/pkg/net/http/server.go:669 (0x64591)
    (*conn).serve: handler.ServeHTTP(w, w.req)
/usr/local/Cellar/go/1.0.3/src/pkg/runtime/proc.c:271 (0x1404b)
    goexit: runtime·goexit(void)
```

**Comment by lukevers** [comment]
May have to do with the number of indexers.

**Comment by alexander-bauer** [comment]
Closed by 239fdb54214bec84eb320a6a10d986da58ee5f95, fixed in v0.8.4.
